src/effects/PanEffect.js

```diff
@@ -65,12 +65,12 @@ class PanEffect extends Effect {
         this.leftGain.gain.setTargetAtTime(
             leftVal,
             this.audioEngine.audioContext.currentTime,
```

Comment:
shouldn't these now have DECAY_WAIT on them too?

Reply:
I was just thinking about that too. Yes they should.